CleanWeather/Helpers/SharedViewController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  SharedViewController.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 16/11/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+class SharedViewController: UIViewController {
+    
+    func presentSpinner(_ state: Bool) {
+        if state {
+            let spinner = SpinnerController()
+            navigationController?.present(spinner, animated: false, completion: nil)
```

Comment:
w zasadzie moglibysmy tutaj dopisac zabezpieczeniiie, bo jak zrobisz presentSpinner(true) 2 razy to ci wywali blad ze probujesz prezentowac kontreoller ktory juz prezentuje cos innego, wiec moze guard navigationController?.presentedViewController == nil